src/components/TypeAhead.svelte

```diff
@@ -2,19 +2,19 @@
   import { onMount } from "svelte";
   import { state as authState } from "../auth.js";
   import { all_docs } from "../couch.js";
+  import Option from "../components/Option.svelte";
```

Comment:
I would move Option.svelte into a Typeahead subdirectory under components. Then this import would be `import Option from "./Typeahead/Option.svelte";`.